### PR TITLE
Add oas endpoints to api submenu

### PR DESCRIPTION
**Pull request by halvorsanden**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/P08pzWFi](https://trello.com/c/P08pzWFi)

**Comment by halvorsanden** [comment]
Azure is still having problems …

Screenshot 2022-02-24 at 15 09 24

**Comment by halvorsanden** [comment]
This will still be hidden in staging and prod, at least until we unhide the page

File: layouts/partials/sidemenuitem.html [hunk 1]
```diff
@@ -37,14 +44,30 @@
...
+
+      {{- with $oasData -}}
+        {{- range .paths -}}
+          {{- range . -}}
+            <li>
+              <a href="#{{ .summary | anchorize }}">
+                <span data-hover="{{ .summary }}"
+                  >{{ .summary }}</span
+                >
+              </a>
```
Comment (Igor-J86) [inline layouts/partials/sidemenuitem.html:57]: Looks like the formatting is somehow fishy here..

Comment (halvorsanden) [reply layouts/partials/sidemenuitem.html:57]: Yes, it’s the autoformat of prettier – just copied what was already done. It works fine though, but looks off. I’ll fix this.

Comment (Igor-J86) [reply layouts/partials/sidemenuitem.html]: Looks good now :)

**Pull request merged**